WowPacketParser/Parsing/Parsers/UpdateHandler.cs

```diff
@@ -2604,6 +2775,14 @@ private static MovementInfo ReadMovementUpdateBlock(Packet packet, WowGuid guid,
                             packet.ReadSingle("Final Spline Orientation", index);
                         else if (splineFlags.HasAnyFlag(SplineFlag.FinalPoint))
                             packet.ReadVector3("Final Spline Coords", index);
+                        else if (splineFlags.HasAnyFlag(SplineFlag.CatmullRom))
+                            packet.ReadSingle("Spline Rotation", index);
+
+                        if (ClientVersion.RemovedInVersion(ClientVersionBuild.V3_1_1_9806))
+                        {
+                            if (splineFlags.HasAnyFlag(SplineFlag.FinalOrientation))
```

Comment:
definitely not correct (i dont even know the structure) - you will attempt to read this twice for builds < 3.1.1

Reply:
Will doublecheck this one. I am taking this info from our old parser, which at least works for these versions.